.changeset/five-islands-happen.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/tp-adapter': patch
+---
+
+Add mapping comment
```

Comment:
I would actually avoid adding a changeset where not actual src code change is made. This creates a new version but it would be exact same version (logic, execution) as the previous one.